buildkite-cli/plan.ps1

```diff
@@ -0,0 +1,18 @@
+$pkg_name="buildkite-cli"
+$pkg_origin="core"
+$pkg_version="0.4.1"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=@("MIT")
+$pkg_description="A command line interface for Buildkite"
+$pkg_deps=@("core/git", "tduffield/buildkite-agent/3.9.1/20190501115102")
```

Comment:
we'll need to get the core/agent released first and update this, correct?

Reply:
Yes. I'll update the PR.